src/sentry/integrations/slack/views/unlink_identity.py

```diff
@@ -58,8 +58,8 @@ def handle(self, request: Request, signed_params: str) -> Response:
 
         try:
             Identity.objects.filter(idp=idp, external_id=params["slack_id"]).delete()
-        except IntegrityError as e:
-            logger.error("slack.unlink.integrity-error", extra=e)
+        except IntegrityError:
+            logger.exception("slack.unlink.integrity-error")
```

Comment:
mypy with django-stubs pointed out that `e` is not a dictionary so this would raise a `TypeError` when formatting the log entry